sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/tests/TimeSeriesInsightsTypesTests.cs

```diff
@@ -21,6 +21,60 @@ public TimeSeriesInsightsTypesTests(bool isAsync)
         {
         }
 
+        [Test]
+        public async Task TimeSeriesInsightsTypeWithNumericVariable_ExpectsError()
+        {
+            // Arrange
+            TimeSeriesInsightsClient client = GetClient();
+            var timeSeriesTypes = new List<TimeSeriesType>();
+            var tsiTypeNamePrefix = "type";
+            var timeSeriesTypesName = Recording.GenerateAlphaNumericId(tsiTypeNamePrefix);
```

Comment:
nit: since it's not implied what the type is for the generated Id (could be Guid, string, etc), it's probably a good idea to change var to string.